rust/azure_iot_operations_connector/src/file_mount_azure_device_registry.rs

```diff
@@ -1,4 +1,270 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-//! Wrapper for Azure Device Registry Client that uses file mount to get names and create/delete notifications.
+//!Azure Device Registry Client that uses file mount to get names and create/delete notifications.
+
+use notify::RecommendedWatcher;
+use std::path::PathBuf;
+use std::sync::{Arc, Mutex};
+use std::time::Duration;
+use tokio::sync::mpsc::UnboundedReceiver;
+use tokio::sync::oneshot;
+
+const ADR_RESOURCES_NAME_MOUNT_PATH: &str = "/etc/akri/config/adr_resources_names";
```

Comment:
Is this always the file path? I thought we'd need to get the file path from an env var (this is fine for now, we can wait to change this in the implementation PR)